samplers/jaegerremote/sampler_remote.go

```diff
@@ -105,7 +104,7 @@ func (s *Sampler) ShouldSample(p trace.SamplingParameters) trace.SamplingResult
 // go-routines it may have started.
 func (s *Sampler) Close() {
 	if swapped := atomic.CompareAndSwapInt64(&s.closed, 0, 1); !swapped {
-		otel.Handle(fmt.Errorf("repeated attempt to close the sampler is ignored"))
+		s.logger.Info("repeated attempt to close the sampler is ignored")
```

Comment:
This changed from an error to an info log message? That does not seem appropriate.

Reply:
I think it's good to use info level here. The close function should only be called once, and multiple calls won't have any effect.

Reply:
If you would like to make a behavior change please do so in a dedicated PR that has supporting and objective motivation.

Otherwise, please limit this PR to scope of adding a logger option to this sampler.